### DISCUSSION TITLE
Add to Scoop package manager

### DISCUSSION THREAD

**Discussion by luciodaou** [f1multiviewer/issue-tracker#206] [Feature requests]
I'd like to suggest adding F1MV to Scoop (https://github.com/ScoopInstaller/Scoop). It is a windows command-line package manager that allows for easy update for those used to deal with prompts/shells. It is basically a windows version of MacOS' homebrew/`brew` or Linux's `apt`, `yum`, `pacman`, amongsts others.

Even though its focus is on open source software, today it contains a lot of closed-source apps through additional repositories.

I believe it's a way to help updating it without the certification hassle mentioned on issue #39 .

Their wiki contains a lot of the info needed to set it up:
https://github.com/ScoopInstaller/Scoop/wiki

**Comment by supersnellehenk** [comment]
Also, winget exists. https://docs.microsoft.com/en-us/windows/package-manager/winget/

**Reply by luciodaou** [reply]
The thing with WinGet is that it's not exactly a package manager, it just installs and uninstalls stuff, there's no actual update management. And as it requires an installer, it falls on the same limitations of #39 regarding a certificate.

**Comment by M0UL** [comment]
chocolaty could also work.

**Comment by jeppeolesen** [comment]
Closing this as wontfix.

Offering official alternative release channels other than the installers on the website is a nice-to-have that is unlikely to happen. Since this ticket was created, the autoupdater has also been integrated in the Windows build, so less need for a package manager solution.
